src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -82,6 +96,52 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
           )
       }
   }
+
+  def getClosureMetadata: Future[Seq[MetadataField]] = {
```

Comment:
As the comment about, maybe move this function into their own service: `CustomMetadataPropertiesService`